frontend/src/app/features/homescreen/blocks/new-features.component.ts

```diff
@@ -33,12 +33,12 @@ import { I18nService } from 'core-app/core/i18n/i18n.service';
 import { imagePath } from 'core-app/shared/helpers/images/path-helper';
 
 // The key used in the I18n files to distinguish between versions.
-const OpVersionI18n = '14_5';
+const OpVersionI18n = '14_6';
 
-const OpReleaseURL = 'https://www.openproject.org/docs/release-notes/14-5-0/';
+const OpReleaseURL = 'https://www.openproject.org/docs/release-notes/14-6-0/';
```

Comment:
Receive 404 error, is it ok? It will be solved later?

Reply:
Yes, this link should be available together with the release. I checked the last teaser PR, it uses the same URL style: https://github.com/opf/openproject/pull/16588/commits/d1bffeba22fa0a045e0908ebc7c15ea5933e73c9